build.gradle

```diff
@@ -69,4 +69,8 @@ shadowJar {
     archiveFileName = 'addressbook.jar'
```

Comment:
Should we change this to `researchroster.jar` as well?

Reply:
Nice catch! Changing right now~